src/energyplus/ForwardTranslator.cpp

```diff
@@ -3196,6 +3201,19 @@ namespace energyplus {
     // result.push_back(IddObjectType::OS_PhotovoltaicPerformance_EquivalentOneDiode);
     // result.push_back(IddObjectType::OS_PhotovoltaicPerformance_Simple);
 
+    // TODO: These should be translated by their ElectricLoadCenter_Distribution, shouldn't they?
```

Comment:
I think yes. especially if these objects do not function on their own. (without the load center)

Reply:
Cleaned merge conflict in https://github.com/NREL/OpenStudio/pull/4194/commits/8268d311234f4a6a71aab4f9d0c860dac2d567b2